index.html

```diff
@@ -1,19 +1,91 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <title>Karma</title>
     <link href='https://fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
     <link rel="stylesheet" href="css/normalize.css">
     <link rel="stylesheet" href="css/style.css">
-    <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
+    <link rel="shortcut icon" type="image/x-icon" href="favicon.ico">
 </head>
+
 <body>
 
     <!-- Add your HTML markup here -->
     <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
     <!-- All the images you need are in the 'img' folder -->
 
+
+    <!-- header part -->
+    <header>
+        <img class="karmaIcon" src="./img/karma-logo.svg" alt="karma's icon">
+        <nav>
+            <ul>
+                <li>
+                    Meet Karma
+                </li>
+                <li>
+                    How it Works
+                </li>
+                <li>
+                    Store
+                </li>
+                <li>
+                    Blog
+                </li>
+                <li>
+                    Help
+                </li>
+                <li>
+                    Login
+                </li>
+            </ul>
+        </nav>
+    </header>
+
+    <main>
+
+        <!-- first section part -->
+        <section class="main-area" style="background-image: url(./img/first-background.jpg);" role="img"
+            aria-label="half of a man in the frame, mam is sitting at a table and working, tablet and karma device on the table">
+            <h3>Introducing Karma</h3>
+            <h4>Bring WiFi with you, everywhere you go.</h4>
+            <button>Learn More</button>
+        </section>
+
+
+        <!-- second section part -->
+        <section class="article-area">
+            <h3>Everyone needs a little Karma.</h3>
+            <div class="article-grid">
+                <img class="article-icon" src="./img/icon-devices.svg" alt="icon of pc, tablet and mobile phone">
+                <img class="article-icon" src="./img/icon-coffee.svg" alt="icon of a coffee mug with steam coming out">
+                <img class="article-icon" src="./img/icon-refill.svg" alt="icon of a gas pump from a refill station">
+                <h4>Internet for all devices</h4>
+                <h4>Boost your productivity</h4>
+                <h4>Pay as You Go</h4>
+            </div>
+        </section>
+
+    </main>
+    <!-- footer part -->
+    <footer>
+        <hr>
+        <h5>Join us on</h5>
+        <div>
+            <img class="media-logo" src="./img/twitter-icon.svg" alt="twitter icon">
```

Comment:
Same as the header, it is best practice to add a tag for each social media link.